app/models/client.rb

```diff
@@ -0,0 +1,3 @@
+class Client < ApplicationRecord
```

Comment:
What is client?

Reply:
owner of the ticket

Reply:
You have `belongs_to :movie` relation here. I believe that: 1) the relation should be with a ticket, not movie ;) 2) `belongs_to` would mean that client can have just a single ticket - I think it should be the other way around

Reply:
True.